doc/client_design.md

```diff
@@ -0,0 +1,129 @@
+# Design Doc: PaddleCloud Client
+
+A Command Line Interface for PaddlePaddle Cloud
+
+---
+
+# Goals:
+
+Developers using PaddlePadle Cloud can use this command-line client for conviniece of manage cloud Deep-Learning jobs, including:
```

Comment:
`conviniece of manage cloud Deep-Learning jobs ` => `managering cloud Deep-Learning jobs conveniently` ?